NEWS.md

```diff
@@ -1,5 +1,15 @@
 # User visible changes for EasyFITS
 
+## Version 0.5.12 (not bumped yet)
+
+- For FITS tables, cell dimensions are mandatory for columns whose cell type is `String`. The user
+must now provide the maximum number of characters for the strings. It avoids ambiguities. When the
+user define a column with cell type `Int` and no dimension, the cell is assumed scalar. But a
+`String` is non-scalar by definition. Previous code, when no dimension provided, was setting
+maximum string length to `1`, which is certainly not what the user meant.
+- In the same topic as the previous bullet: trying to add a string that is too long for the cell
+maximum now throws an error, instead of truncating. It avoids writing wrong data in the file.
+
```

Comment:
Please shorten this description (otherwise no one will really read it) and, since the 2 are related, make a single bullet with 2 sub-bullets. For example:

- Columns of strings in FITS tables:
  - An error is raised when attempting to write strings longer than the maximal length in a column of strings. The previous behavior was to (silently) truncate the strings.
  - When cell dimensions are not specified in a column definition, it is assumed that a single value is stored by each cell of that column. This is inappropriate for strings for which the default amounts to storing a single character per cell. Cell dimensions are therefore now mandatory for defining columns of strings in a table HDU. Note that this is the simplest way to specify the maximum length of the strings.